cpubars.c

```diff
@@ -22,6 +22,14 @@
 #include <curses.h>
 #include <term.h>
 
+#ifdef __FreeBSD__
+#define PROCPATH "/compat/linux/proc"
```

Comment:
This is fine, but what would you think of simply trying both paths rather than switching on the OS?